### PR TITLE
Remove BeforeFeature function from crc integration test suite

### PR DESCRIPTION
**Pull request by praveenkumar**
In BeforeFeature, bundle is downloaded and testdata file is moved
to the test run directory. Both can be part of BeforeSuite.

### CODE REVIEW

**Review by guillaumerose — APPROVED** [review]

**Comment by praveenkumar** [comment]
retest this please

File: test/integration/crcsuite/crcsuite.go [hunk 1]
```diff
@@ -114,6 +114,33 @@ func FeatureContext(s *godog.Suite) {
...
+				fmt.Println("Using bundle:", bundleLocation)
+			} else {
+				fmt.Println("Using existing bundle:", bundleLocation)
+			}
+		}
+	})
+
+	s.BeforeScenario(func(pickle *messages.Pickle) {
+		// copy data/config files to test dir
+		err := CopyFilesToTestDir()
```
Comment (jsliacan) [inline test/integration/crcsuite/crcsuite.go:140]: Should this be in `BeforeSuite`? It copies [this folder's](https://github.com/code-ready/crc/tree/master/test/testdata) contents to the execution folder (i.e. `crc/test/integration/out/run`). The execution folder should stay put throughout all Features - the previous change with bundle download relies on this iirc?

Comment (praveenkumar) [reply test/integration/crcsuite/crcsuite.go:140]: `BeforeSuite` is deprecated and we should limit using deprecated features.

Comment (jsliacan) [reply test/integration/crcsuite/crcsuite.go:140]: I thought `BeforeFeature` was deprecated, thread [here](https://www.gitmemory.com/issue/cucumber/godog/335/665641230).

Comment (praveenkumar) [reply test/integration/crcsuite/crcsuite.go:140]: Ah right so `BeforeFeature` is deprecated for bundle I changed the logic which can't be done for these testdata files.

**Pull request merged**